test/integration/targets/openssl_publickey/tasks/main.yml

```diff
@@ -3,11 +3,48 @@
       openssl_privatekey:
         path: '{{ output_dir }}/privatekey.pem'
 
-    - name: Generate publickey
+    - name: Generate publickey - PEM format
       openssl_publickey:
         path: '{{ output_dir }}/publickey.pub'
         privatekey_path: '{{ output_dir }}/privatekey.pem'
 
+    - name: Generate publickey - OpenSSH format
+      openssl_publickey:
+        path: '{{ output_dir }}/publickey-ssh.pub'
+        privatekey_path: '{{ output_dir }}/privatekey.pem'
+        format: OpenSSH
+      when: cryptography_version.stdout|version_compare('1.4.0', '>=')
```

Comment:
Please add a comment explaining why the cryptography version filtering is necessary. If the reason is the same for all occurrences, a single explanation for all occurrences is fine.